backend/controller/observability/async_calls.go

```diff
@@ -24,10 +25,11 @@ const (
 )
 
 type AsyncCallMetrics struct {
-	meter     metric.Meter
-	acquired  metric.Int64Counter
-	executed  metric.Int64Counter
-	completed metric.Int64Counter
+	meter        metric.Meter
```

Comment:
nit: Similar note to others, we might not need to save a reference to the `meter` here.

Reply:
ah sweet, removed